style.css

```diff
@@ -21,6 +21,75 @@ body {
   display: none;
 }
 
+.popup-buttons {
+  display: flex;
+}
+
+.popup-default {
+  display: none;
+}
+
+.show {
+  display: block;
+  position: fixed;
+  top: 10px;
+  bottom: 10px;
+  border-radius: 0.5rem;
+  background: white;
+  left: 2rem;
+  right: 2rem;
+  padding: 1rem;
+}
```

Comment:
- [ ] Kindly the modal can't be scrolled so I can't see the rest of the modal. Kindly make it scrolling to be like the Figma design.

![image](https://user-images.githubusercontent.com/35535130/156489366-71923cd8-6fae-40dc-9100-a2dcc1a58d24.png)

Reply:
added